sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/TestHelper.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.appconfiguration;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.provider.Arguments;
+
+import static com.azure.core.test.TestBase.getHttpClients;
+
+class TestHelper {
+    static final String DISPLAY_NAME_WITH_ARGUMENTS = "{displayName} with [{arguments}]";
+    private static final String AZURE_TEST_SERVICE_VERSIONS = "AZURE_APPCONFIG_SERVICE_VERSIONS";
```

Comment:
Name the variable and value as `AZURE_APPCONFIG_TEST_SERVICE_VERSIONS` as we don't want it to be mistaken for some non-test related env variable. 
```suggestion
    private static final String AZURE_APPCONFIG_TEST_SERVICE_VERSIONS = "AZURE_APPCONFIG_TEST_SERVICE_VERSIONS";
```